k8s/backend-all.yaml

```
@@ -11,6 +11,38 @@ spec:
 
 ---
 
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: challenge-flutter-go-backend-pv
+  namespace: challenge-flutter-go
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 5Gi
+  volumeMode: Filesystem
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: "/mnt/challenge-flutter-go-backend-pv/uploads"
+
+---
+
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: challenge-flutter-go-backend-pvc
+  namespace: challenge-flutter-go
+spec:
+  storageClassName: manual
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
```

```suggestion
      storage: 20Gi
```

C'est peut etre plus acceptable pour de l'upload de fichier